internal/operator/internal/operator_installer.go

```diff
@@ -18,10 +18,20 @@ import (
 	"context"
 	"fmt"
 
-	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+	v1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	"github.com/operator-framework/operator-sdk/internal/operator"
+	"github.com/operator-framework/operator-sdk/internal/util/k8sutil"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
 
-	"github.com/operator-framework/operator-sdk/internal/operator"
+// InstallPlanApproval - type of InstallPlan approval for the subscription
+type InstallPlanApproval = string
+
+const (
+	// ManualApproval is the manual install plan approval
+	ManualApproval InstallPlanApproval = "Manual"
+	// AutomaticApproval is the automatic install plan approval
+	AutomaticApproval InstallPlanApproval = "Automatic"
```

Comment:
Why not use [`v1alpha1.Approval`](https://github.com/operator-framework/api/blob/master/pkg/operators/v1alpha1/installplan_types.go#L21-L22) directly?